devstart.sh

```diff
@@ -1,24 +1,12 @@
 #!/bin/bash
 
-pushd "${SOURCE_DIR}/react-spark/example";
```

Comment:
The example app was causing problems so I removed it from the devstart. I removed it from the devsite too.

I didn't go so far as to remove the code completely though. I can if we feel it's necessary.